### ISSUE TITLE
Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by haenf** [muonw/powertable#15]
I try to use PowerTable in a Sveltekit project (sveltejs/kit@1.0.0-next.344). I imported the PowerTable setup in the svelte page as you described in the Quickstart. Then when access the page I get a 500 internal error, with the message "Cannot read properties of undefined (reading 'length')". What can I do to make things work?

**Comment by haenf** [comment]
The problem is in the _...@muonw/powertable/package/dist/power-table.css_ : there is no such css file in that directory. So I copied the _power-table.scss_ from your Github pages and put that in the _dist_ directory. My PowerTable page then looks like this below. Is that correct?

![PowerTable_screen](https://user-images.githubusercontent.com/17638039/215852760-c3cfe0a8-05af-4d5c-b84a-c1620ab6415a.png)

**Comment by muonw-public** [comment]
Thank you for reporting the problem and sharing your workaround. The screenshot you've shared shows the expected output 👍. Styling in the current version is messy and will be improved. In the meantime, you can make the following changes to make the table look better.

1. Install [Mascara](https://github.com/muonw/mascara):
```
npm i muonw/mascara
```

2. Replace the content of your <style> tag with the following:

```svelte
<style lang="scss" global>
@import '../../node_modules/@muonw/mascara/package/styles/index.scss';
@import '../../node_modules/@muonw/powertable/package/styles/power-table.scss';

/* To make search and filter text boxes smaller */
.MuonW.PowerTable {
    tr[data-name=filters-tr] {
        input{
            @extend .compact;
        }
    }
    tr[data-name=filters-tr] {
        input{
            @extend .compact;
        }
    }
    div[data-name=search-container], div[data-name=edit-block] {
        label{
            @extend .embedded;
            & > span, &:focus-within > span {
                @extend .label-text;
            }
        }
    }
    div[data-name=edit-block] {
        button[data-name=edit-submit] {
            margin-top: 8px;
        }
    }
}
</style>
```

**Comment by haenf** [comment]
Thank you for your quick reply! I got it all working now. It looks like a very nice application and it does a lot of the things I was looking for. So I have a back-end with MongoDB and Julia, and a front-end with Sveltekit and - now - PowerTable. The functionality is most important for me right now, and the styling will come (I'm good at design, but I see I have to learn SCSS... :- ).

**Comment by muonw** [comment]
Fixed in [v1.0.0](https://github.com/muonw/powertable/releases/tag/v1.0.0)
